src/Selenium2Library/keywords/_element.py

```diff
@@ -599,44 +599,51 @@ def page_should_not_contain_image(self, locator, message='', loglevel='INFO'):
 
     # Public, xpath
 
-    def get_matching_xpath_count(self, xpath):
+    def get_matching_xpath_count(self, xpath_):
         """Returns number of elements matching `xpath`
 
-        One should not use the xpath= prefix for 'xpath'. XPath is assumed.
+        As of release 1.8 `Get Matching Xpath Count` allows the
+        argument `xpath` to start with the xpath= locator strategy.
 
-        Correct:
+        Example:
         | count = | Get Matching Xpath Count | //div[@id='sales-pop']
-        Incorrect:
         | count = | Get Matching Xpath Count | xpath=//div[@id='sales-pop']
 
         If you wish to assert the number of matching elements, use
         `Xpath Should Match X Times`.
         """
-        count = len(self._element_find("xpath=" + xpath, False, False))
+        if not xpath_.lower().lstrip().startswith('xpath='):
+            xpath_ = "xpath=" + xpath_
+        
+        count = len(self._element_find(xpath_, False, False))
         return str(count)
 
-    def xpath_should_match_x_times(self, xpath, expected_xpath_count, message='', loglevel='INFO'):
-        """Verifies that the page contains the given number of elements located by the given `xpath`.
+    def xpath_should_match_x_times(self, xpath_, expected_xpath_count, message='', loglevel='INFO'):
+        """Verifies that the page contains the given number of elements located by the given `xpath_`.
 
-        One should not use the xpath= prefix for 'xpath'. XPath is assumed.
+        As of release 1.8 `Xpath Should Match X Times` allows the
+        argument `xpath_` to start with the xpath= locator strategy.
 
-        Correct:
+        Example:
         | Xpath Should Match X Times | //div[@id='sales-pop'] | 1
-        Incorrect:
         | Xpath Should Match X Times | xpath=//div[@id='sales-pop'] | 1
 
         See `Page Should Contain Element` for explanation about `message` and
         `loglevel` arguments.
         """
-        actual_xpath_count = len(self._element_find("xpath=" + xpath, False, False))
+        if not xpath_.lower().lstrip().startswith('xpath='):
```

Comment:
Just to think we are that close to support locators with the strategy in, I'm getting sick of removing it from my locator vars to verify the correct number of elements are present :smile:  
All joking aside, what is needed to get the PR merged? This will be a really helpful improvement; thanks a lot in advance :+1: